admin_app/src/app/content/edit/page.tsx

```diff
@@ -143,6 +167,30 @@ const ContentBox = ({
     setIsSaved(false);
   };
 
+  const handleLanguageSelect = (language_id: number) => {
+    setContent((prevContent) => {
+      if (prevContent === null) {
+        // If prevContent is null, initialize it as needed
+        return {
```

Comment:
Isn't this a type? Can we initialize from there? We will add another prop to the type one day and forget to update this.